spec/vsphere_connection_handling_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require 'chef/knife/base_vsphere_command'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.